ResidentInformationApi/V1/Gateways/AcademyInformationGateWay.cs

```diff
@@ -12,19 +12,21 @@ namespace ResidentInformationApi.V1.Gateways
     public class AcademyInformationGateway : IAcademyInformationGateway
     {
         private readonly HttpClient _client;
-        private readonly string _baseUrl;
-
         public AcademyInformationGateway(HttpClient client)
         {
             _client = client;
-            _baseUrl = Environment.GetEnvironmentVariable("ACADEMY_API_ENDPOINT");
+            //_baseUrl = Environment.GetEnvironmentVariable("ACADEMY_API_ENDPOINT");
         }
         public async Task<List<AcademyClaimantInformation>> GetClaimantInformation(ResidentQueryParam rqp)
         {
             var rqpString = DictionaryBuilder.BuildQueryDictionary(rqp);
-            var builder = new UriBuilder(_baseUrl);
+            var builder = new UriBuilder();
             builder.Query = rqpString;
-            var response = await _client.PostAsync(builder.Uri, null).ConfigureAwait(true);
+            var response = await _client.GetAsync(new Uri(builder.Query, UriKind.Relative)).ConfigureAwait(true);
+
+            //throw exception if not 200
+            response.EnsureSuccessStatusCode();
```

Comment:
These exceptions are new functionality and so should be tested

Reply:
Will need some pointers on testing this

Reply:
If you are expecting it to throw then you can unit test this but getting the mock API to return a 400 (or another failing message) and then assert that the class throws

Reply:
ok